packages/parcel-optimizer-html-inline-css/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# @compiled/parcel-optimizer-html-inline-css
```

Comment:
I am not too sure about the name of this package. Its job is to retrieves asset.meta.styleRules, optimizes the rules, and inject the rules to HTML. Please let me know if you can think of a better name.